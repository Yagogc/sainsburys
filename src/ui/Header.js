import styled from "styled-components";

export const Header = styled.header`
  background: #f90;
  padding: 10px;
  margin-bottom: 15px;
  text-align: center;
  box-shadow: 0 1px 3px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24);
`;

export const HeaderTitle = styled.h1`
  color: #fff;
`;
