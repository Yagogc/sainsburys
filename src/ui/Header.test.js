import React from "react";
import renderer from "react-test-renderer";
import { Header, HeaderTitle } from "./Header";

test("UI - Header:", () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});

test("UI - HeaderTitle:", () => {
  const tree = renderer.create(<HeaderTitle />).toJSON();
  expect(tree).toMatchSnapshot();
});
