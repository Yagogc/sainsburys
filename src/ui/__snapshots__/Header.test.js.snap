// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI - Header: 1`] = `
.c0 {
  background: #f90;
  padding: 10px;
  height: 58px;
  text-align: center;
  box-shadow: 0 1px 3px rgba(0,0,0,0.12),0 1px 2px rgba(0,0,0,0.24);
}

<header
  className="c0"
/>
`;

exports[`UI - HeaderTitle: 1`] = `
.c0 {
  color: #fff;
}

<h1
  className="c0"
/>
`;
